modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/disruptor/inbound/InboundEventContainer.java

```diff
@@ -194,6 +213,9 @@ public void updateState() throws AndesException {
             case TRANSACTION_CLOSE_EVENT:
                 getTransactionEvent().updateState();
                 break;
+            case DTX_COMMIT_EVENT:
+                getDtxBranch().updateState();
+                break;
             case PUBLISHER_RECOVERY_EVENT:
                 executeRecoveryEvent();
                 break;
```

Comment:
No. dtx.start and dtx.end are handled at the AMQP transport level. No need to expose them to andes-core.